### ISSUE TITLE
Editors

### ISSUE THREAD

**Issue by macMikey** [macMikey/LCB-missing-manual#1]
Editors are discussed in [Property Definitions Readme](https://github.com/livecode/livecode-ide/blob/develop/Toolset/resources/supporting_files/property_definitions/PROPERTY_DEFINITIONS_README.md#editor), and are supposed to appear in [Extending LiveCode](https://livecode.com/topic/extending-livecode-guide/), but they do not.

**Comment by bwmilby** [comment]
Those are a part of the IDE.  Here is the path from the open source repository:
`.../livecode/ide/Toolset/palettes/inspector/editors/`
You should also be able to find them in your local install of LC.
`/Applications/LiveCode/LiveCode 9.6.9 (rc 1).app/Contents/Tools/Toolset/palettes/inspector/editors/`
I agree that they should be documented somewhere.  Looking at the names there, most that you would use are probably fairly evident.

**Comment by macMikey** [comment]
Right. I think I made a note of that in the property metadata wiki article. If I didn't, it certainly belongs in the reference section.

**Comment by macMikey** [comment]
also, i see there is a `subeditor` metadata that is not documented.

**Comment by macMikey** [comment]
https://livecode.com/topic/hacking-the-ide-properties-elanor-buchanan-2pm/
I'm adding a [Property Editors](https://github.com/macMikey/LCB-missing-manual/wiki/Property-Editors) page based on [elanor's talk](https://livecode.com/topic/hacking-the-ide-properties-elanor-buchanan-2pm/)

**Comment by macMikey** [comment]
However, it's going to take a LOT of work to make it complete...
